eZ/Publish/Core/Repository/Values/User/Policy.php

```diff
@@ -33,4 +33,19 @@ public function getLimitations()
     {
         return $this->limitations;
     }
+
+    /**
+     * Return true if the scope of current Policy object is more narrow than the scope of $policy.
+     *
+     * @param \eZ\Publish\Core\Repository\Values\User\Policy $policy
+     * @return bool
+     */
+    public function isNarrow(Policy $policy)
+    {
+        return
+            !empty($this->limitations) && empty($policy->getLimitations()) &&
+            ($this->module === $policy->module || $policy->module === '*') &&
+            ($this->function === $policy->function || $policy->function === '*')
+        ;
```

Comment:
To simplify [this code](https://github.com/alongosz/ezpublish-kernel/blob/d912295e74e37ecbeb318b7d9f6953076836f243/eZ/Publish/Core/Repository/Repository.php#L481), I've decided to move checking if policy is more narrow to `\eZ\Publish\Core\Repository\Values\User\Policy` `ValueObject`. It's kind-of comparison method, so IMHO it doesn't break ValueObject pattern.

WDYT @andrerom, @pspanja?